tests/system/feature_store/test_spark_engine.py

```diff
@@ -80,3 +86,108 @@ def test_basic_remote_spark_ingest(self):
             spark_context=self.spark_service,
             run_config=fs.RunConfig(local=False),
         )
+
+    @pytest.mark.parametrize("partitioned", [True, False])
+    def test_schedule_on_filtered_by_time(self, partitioned):
+        name = f"sched-time-{str(partitioned)}"
```

Comment:
Isn't it the same name as in the non-spark tests? Maybe it's better to add "spark" to the name?

Reply:
Actually, it's a different project, no need to rename.
Is it the same code? Maybe we can add a utils file and import it from there in both test files? (Not critical)